### ISSUE TITLE
face are not color correctly

### ISSUE THREAD

**Issue by sophietheis** [DamCB/tyssue#84]
This version of draw face doesn't color face according to a condition like area. It seems to color face randomly. 

```
def draw_face(sheet, coords, ax, **draw_spec_kw):
    """Draws epithelial sheet polygonal faces in matplotlib
    Keyword values can be specified at the element
    level as columns of the sheet.face_df
    """

    draw_spec = sheet_spec()['face']
    draw_spec.update(**draw_spec_kw)
    collection_specs = parse_face_specs(draw_spec, sheet)

    if 'visible' in sheet.face_df.columns:
        edges = sheet.edge_df[sheet.upcast_face(sheet.face_df['visible'])].index
        sheet = get_sub_eptm(sheet, edges)
        color = collection_specs['facecolors']
        if isinstance(color, np.ndarray):
            faces = sheet.face_df['face_o'].values.astype(np.uint32)
            collection_specs['facecolors'] = color.take(faces, axis=0)

    polys = sheet.face_polygons(coords)
    polys = polys.reindex(sheet.face_df.sort_values('z').index)
    p = PolyCollection(polys, closed=True, **collection_specs)
    ax.add_collection(p)
    return ax
```

Color face according to a condition works find with the oldest version of draw_face : 
```
def draw_face(sheet, coords, ax, **draw_spec_kw):
    """Draws epithelial sheet polygonal faces in matplotlib
    Keyword values can be specified at the element
    level as columns of the sheet.face_df
    """
    draw_spec = sheet_spec()['face']
    draw_spec.update(**draw_spec_kw)
    polys = sheet.face_polygons(coords)
    patches = []
    for idx, poly in polys.items():
        patch = Polygon(poly,
                        fill=True,
                        closed=True)
        patches.append(patch)
    collection_specs = parse_face_specs(draw_spec, sheet)
    p = PatchCollection(patches, False,
                        **collection_specs)
    ax.add_collection(p)
    return ax
```

**Comment by glyg** [comment]
Oh I think I get where the error is, the  `polys = polys.reindex(sheet.face_df.sort_values('z').index)` line changes the polygon order, but not the color order - I'm not sure there's a simple fix, maybe it's enough to comment out the line  `polys = polys.reindex(sheet.face_df.sort_values('z').index)` for now.

**Comment by glyg** [comment]
it's not good anyway as this 'z' coord is not necessarily the good coordinates to sort over.

**Comment by glyg** [comment]
fixed by #85
